counter-sink/src/main/java/org/springframework/cloud/stream/module/metrics/CounterSinkConfiguration.java

```diff
@@ -41,17 +36,6 @@
 
     private EvaluationContext evaluationContext;
 
-    @Bean
-    public CounterService counterService(RedisMetricRepository redisMetricRepository) {
```

Comment:
I assume a `CounterService` is auto-instantiated by having the metrics-actuator on the classpath?... and it gets wired with whatever `MetricRepository` has been configured?